tests/unit/moduleapi/dryrun.tcl

```diff
@@ -0,0 +1,36 @@
+set testmodule [file normalize tests/modules/dryrun.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+
+    test {test Dry Run - OK OOM} {
+        set val [r get x]
+        assert_equal [r dryrun set x 5] OK
+        assert_equal [r get x] $val
+    }
+
+    test {test Dry Run - OK ACL} {
+        set val [r get x]
+        assert_equal [r dryrun set x 5] OK
+        assert_equal [r get x] $val
+    }
```

Comment:
where's the ACL setup here?
it's the same test (successful) as before, let's drop it

Reply:
ok